### PR TITLE
@orta: Updates CardFlight SDK for new reader compatibility

### PR DESCRIPTION
**Pull request by ashfurrow**
Fairly straightforward, will comment inline with any gotchas.

### CODE REVIEW

File: Kiosk/App/CardHandler.swift [hunk 1]
```diff
@@ -54,10 +54,14 @@ class CardHandler: NSObject, CFTReaderDelegate {
             logger.log("CardReader got a response it cannot handle")
 
 
-            reader.beginSwipeWithMessage(nil);
+            reader.beginSwipe();
         }
     }
 
+    func transactionResult(charge: CFTCharge!, withError error: NSError!) {
```
Comment (ashfurrow) [inline Kiosk/App/CardHandler.swift]: This is a new, required method. Event though we don't do any transactions, we still need to implement it.

**Comment by orta** [comment]
well, that was easy enough

**Comment by ashfurrow** [comment]
Not quite :stuck_out_tongue: Still need to make sure the old readers work with the new SDK...

**Comment by ashfurrow** [comment]
Currently unable to use older card readers. Investigating now.

**Comment by orta** [comment]
Don't forget there's always a "email the vendor" option

**Comment by ashfurrow** [comment]
Opening an GH issue now.

**Comment by ashfurrow** [comment]
OK, I've pulled in the latest changes from master. Still no luck with CardFlight's SDK working with older card readers – I've asked them to re-open my issue. Hopefully won't be too long and we can merge this.

**Comment by ashfurrow** [comment]
@orta This is now ready for review; turns out I had a broken card reader.

**Comment by orta** [comment]
haha

**Comment by ashfurrow** [comment]
Restarted CI – got the intermittent failure.

**Comment by orta** [comment]
:+1:

**Comment by ashfurrow** [comment]
:tada:

**Pull request merged**